install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -521,16 +521,11 @@ data:
   auto-direct-node-routes: {{ .Values.autoDirectNodeRoutes | quote }}
 
 {{- if .Values.bandwidthManager }}
-{{- if typeIs "bool" .Values.bandwidthManager }}
-  # DEPRECATED: this block should be removed in 1.13
-  enable-bandwidth-manager: {{ .Values.bandwidthManager | quote }}
```

Comment:
It seems this change is unrelated to the PR

Reply:
this is  clean up I mentioned in the commit message